code/game/objects/items/weapons/mine.dm

```diff
@@ -6,13 +6,14 @@
 	w_class = ITEM_SIZE_BULKY
 	matter = list(MATERIAL_STEEL = 30)
 	matter_reagents = list("fuel" = 40)
+	layer = ABOVE_MOB_LAYER //should fix all layering problems? or am i crazy stupid and understood it wrong
```

Comment:
Mines above mobs is just wrong.

Reply:
sure can change it too
above turf then?